simcon_project/templates/student_settings_view.html

```diff
@@ -11,25 +14,16 @@
     <style type="text/css">
     form {display:inline; margin:0px; padding:0px;}
     </style>
-    
-
     <title>
     {% block title %}Simulated Conversations | Settings{% endblock %}
     </title>
-    
     {% block extrahead %} {% endblock %}
   </head>
-
   <body>
     <div class="container">
       <h1> This is the Settings Page</h1>
-        {% block content %}
```

Comment:
Won't we need this?

Reply:
we have the ones encasing the whole html